### DISCUSSION TITLE
How do I tell what Homepage Version do I have?

### DISCUSSION THREAD

**Discussion by Clyde6790pGIT** [tersiswilvin/Roblox-Legacy-Old-Theme#61] [General]
There are like 7 versions available and I can't tell which one it is, I've tried like all of them and most are even more broken or just kind of better.

**Comment by ghost** [comment]
You might have the same version of me, or problem. Me and @tersiswilvin were talking about this, saying we should just wait until we see what version Roblox settles on. For more context you may want to see this recording showing the message history:
https://github.com/tersiswilvin/Roblox-2019-Old-Theme/assets/77752310/4f81df96-af1e-4e69-b6b9-69672dcb35cb
